Service/src/util/db_connection.py

```diff
@@ -2,7 +2,7 @@
 import os
```

Comment:
See how file path is configured here...replicate it for your iextoken

Reply:
I tried importing this function from this file and it would call the global var from the wrong config file. I'll just rewrite the function in the charts.py

Reply:
Followed the same model and fixed it